src/client/app/styles/modalStyle.tsx

```diff
@@ -2,20 +2,128 @@
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import React from 'react';
+
+export const bottomSpace: React.CSSProperties = {
+	paddingBottom: '10px'
+};
+
+export const checkboxStyle: React.CSSProperties = {
+	marginRight: '10px'
+};
+
+export const divTopBottomPadding: React.CSSProperties = {
+	paddingTop: '0px',
+	paddingBottom: '15px'
+};
+
+export const footerStyle: React.CSSProperties = {
+	borderTop: '1px #e1e4e8 solid',
+	padding: '10px 0px',
+	textAlign: 'center',
+	width: '100%'
+};
+
+export const fullSizeContainer: React.CSSProperties = {
+	width: '100%',
+	height: '100%',
+	minHeight: '700px'
+};
+
 export const formInputStyle: React.CSSProperties = {
 	paddingBottom: '5px'
 };
 
+export const labelStyle: React.CSSProperties = {
+	fontWeight: 'bold',
+	margin: 0
+};
+
+export const largeTitleStyle: React.CSSProperties = {
+	display: 'inline-block'
+};
+
+
+// TODO Styling for the component, may need to be converted into .css files
+export const pillContainer: React.CSSProperties = {
+	display: 'flex',
+	justifyContent: 'space-between',
+	margin: '0px',
+	padding: '0px',
+	minHeight: '100px',
+	maxHeight: '100px'
+};
+
+export const pillBoxLabel: React.CSSProperties = {
+	alignItems: 'start',
+	textAlign: 'left',
+	margin: '0px',
+	padding: '0px'
+};
+
+export const pillBox: React.CSSProperties = {
+	display: 'flex',
+	flexDirection: 'column',
+	justifyContent: 'left',
+	width: '45%',
+	maxWidth: '45%',
+	margin: '0px',
+	padding: '0px'
+};
+
+export const pills: React.CSSProperties = {
+	display: 'flex',
+	flexWrap: 'wrap',
+	justifyContent: 'left',
+	overflowY: 'auto',
+	maxHeight: '120px',
+	paddingRight: '8px'
+};
+
+export const pill: React.CSSProperties = {
```

Comment:
A small point but I think pill comes first alphabetically since it is shorter than other pill strings.